package/MDAnalysis/analysis/base.py

```diff
@@ -411,6 +427,8 @@ def run(self, start=None, stop=None, step=None, frames=None,
                            step=step, frames=frames)
         logger.info("Starting preparation")
         self._prepare()
+        logger.info("Starting analysis loop over %d trajectory frames",
+                    self.n_frames)
```

Comment:
We announce frame selection, preparation and conclusion but not the main analysis loop. Adding this logger call makes it visible in a log file and allows reasonable ad-hoc benchmarking because the number of frames analyzed is also printed: one can calculate `((time of logger call conclusions) - (time of call starting analysis))/n_frames`.